ui/components/PromptComponent.js

```diff
@@ -64,14 +64,17 @@ const styles = (theme) => ({
   },
 });
 
-const PromptActionButton = styled(Button)(({ theme, promptVariant }) => ({
+const PromptActionButton = styled(Button)(({ theme }) => ({
   margin: theme.spacing(0.5),
   padding: theme.spacing(1),
   borderRadius: 5,
-  backgroundColor: theme.palette.secondary[promptVariant || 'primary'],
+  backgroundColor: theme.palette.type === 'dark' ? '#00B39F' : '#607d8b',
```

Comment:
@vaibhavpnimkar there should be a theme color defined for this, we are using so many place 🤔